DashKitUI/Views/DesignElements/RoundedCard.swift

```diff
@@ -0,0 +1,130 @@
+//
+//  RoundedCard.swift
+//  SectionedUIPlaygroundApp
+//
+//  Created by Pete Schwamb on 2/9/21.
+//
+
+import SwiftUI
+
+struct RoundedCardTitle: View {
+    var title: String
+    
+    init(_ title: String) {
+        self.title = title
+    }
+    
+    var body: some View {
+        Text(title)
+            .font(.headline)
+            .foregroundColor(.primary)
+    }
+}
+
+struct RoundedCardFooter: View {
+    var text: String
+    
+    init(_ text: String) {
+        self.text = text
+    }
+    
+    var body: some View {
+        Text(text)
+            .font(.caption)
+            .fixedSize(horizontal: false, vertical: /*@START_MENU_TOKEN@*/true/*@END_MENU_TOKEN@*/)
```

Comment:
[nit] What is `/*@START_MENU_TOKEN@*/true/*@END_MENU_TOKEN@*/`?

Reply:
Ah, that's xcode placeholder text. Will remove.